tests/test_esm_runscripts/test_filedicts.py

```diff
@@ -95,3 +103,11 @@ def test_cp(fs):
     esm_runscripts.filedicts.SimulationFile.cp(source, target)
 
     assert os.path.exists(target)
+
+
+def test_resolve_file_movements(config):
+    # act
+    config = resolve_file_movements(config)
```

Comment:
I would rather call the function via `module.func_name`, as it is done in other tests.

Reply:
agreed

Reply:
What about this? `import esm_runscripts.filedicts as filedicts`